src/lib/rng/stateful_rng/stateful_rng.h

```diff
@@ -72,7 +72,11 @@ class BOTAN_PUBLIC_API(2,0) Stateful_RNG : public RandomNumberGenerator
       * of the length of the input or the current seeded state of
       * the RNG.
       */
-      void initialize_with(const uint8_t input[], size_t length);
+      void initialize_with(std::span<const uint8_t> input);
+      void initialize_with(const uint8_t input[], size_t length)
+         {
+         this->initialize_with(std::span(input, length));
+         }
```

Comment:
This might be another instance of the virtual dispatch issue [discussed here](https://github.com/randombit/botan/pull/3317). Probably we want to also add a protected interface with another name for good measure.